SPEC.md

```diff
@@ -868,7 +868,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
 * **acVersion** (string, required) represents the version of the schema spec (must be in [semver](http://semver.org/) format)
 * **acKind** (string, required) must be set to "PodManifest"
 * **apps** (list of objects, required) list of apps that will execute inside of this pod. Each app object has the following set of key-value pairs:
-    * **name** (string, optional) name of the app (restricted to AC Name formatting)
+    * **name** (string, required) name of the app (restricted to AC Name formatting). This is used to identify an app within a pod, and hence MUST be unique within the list of apps. This may be different from the name of the underlying image (see below).
```

Comment:
Is it clear to a new reader why this isn't just the image name? Add comment that you can run multiple instances (apps) using the same image in a pod?